neo/IO/Helper.cs

```diff
@@ -129,11 +129,39 @@ public static T[] ReadSerializableArray<T>(this BinaryReader reader, int max = 0
             return array;
         }
 
+        public static uint[] ReadSerializableArray(this BinaryReader reader, int max = 0x1000000)
+        {
+            uint[] array = new uint[reader.ReadVarInt((ulong) max)];
+            for (int i = 0; i < array.Length; i++) array[i] = reader.ReadUInt32();
+            return array;
+        }
+
         public static byte[] ReadVarBytes(this BinaryReader reader, int max = 0x1000000)
         {
             return reader.ReadBytes((int)reader.ReadVarInt((ulong)max));
         }
 
+        public static byte[][] ReadArrayOfVarBytesSupportingNull(this BinaryReader reader, int maxItems=255, int maxItemLen = 1024)
```

Comment:
`ReadVarBytesArray`?

Reply:
`ReadVarBytesArray` is fine, accepting your suggestion.